index.html

```diff
@@ -0,0 +1,240 @@
+                                Sponsor
+                            </a>
+                        </li>
+                        <li>
+                            <a href="">
+                                News
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <div>
+                    <ul>
+                        <li>
+                            <a href="index.html">
+                                CC Campaign
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <main>
+        <section class="event-info">
+            <h1>Hello All Aspiring Graduates!</h1>
+            <div>
+                <h2>
+                    ACCRA EDU-FAIR 2022
+                </h2>
+            </div>
+            <div class="event-para">
+                <p class="event-para1">
+                    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Labore, numquam. Adipisci labore possimus atque recusandae excepturi porro incidunt tenetur distinctio.
+                </p>
+            </div>
+            <div>
+                <h3 class="event-date">
+                    EVENT IS SCHEDULED FOR TWO DAYS: 2022-05-24 ~ 25
+                </h3>
+                <p class="event-para2">@ Accra International Conference Center</p>
+            </div>
+        </section>
+        <section class="program-schedule">
+            <h3 class="program-heading">Main Program</h3>
+            <span class="underline"></span>
+            <div class="program-outline">
+                <ul class="program">
+                    <li>
+                        <img src="images/icon1.png" alt="">
+                    </li>
+                    <li>
+                        <h3>Lorem ipsum</h3>
+                    </li>
+                    <li>
+                        <p>Lorem ipsum dolor sit amet consectetur.</p>
+                    </li>
+                </ul>
+                <ul class="program">
+                    <li>
+                        <img src="images/icon3.png" alt="">
+                    </li>
+                    <li>
+                        <h3>Lorem ipsum</h3>
+                    </li>
+                    <li>
+                        <p>Lorem ipsum dolor sit amet consectetur.</p>
+                    </li>
+                </ul>
+                <ul class="program">
+                    <li>
+                        <img src="images/icon4.png" alt="">
+                    </li>
+                    <li>
+                        <h3>Lorem ipsum</h3>
+                    </li>
+                    <li>
+                        <p>Lorem ipsum dolor sit amet consectetur.</p>
+                    </li>
+                </ul>
+                <ul class="program">
+                    <li>
+                        <img src="images/icon2.png" alt="">
```

Comment:
- [ ] Kindly make sure that you write all the alt attributes for images. Please don't leave it blank instead describe what the image is.